server/DAO/MealtimeDAO.js

```diff
@@ -0,0 +1,19 @@
+const {MealTimes} = require('../models/Identifier')
+
+class MealtimeDAO {
+
+
+getAllMealtimes = () => MealTimes
+
+getMealtime = (type) => {
+    let mealtimeFound
+    MealTimes.forEach(ele => {
+        if(ele.names.includes(type.toLowerCase())){
+              mealtimeFound = ele.value
+        }
+    })
+    return mealtimeFound ? [{value: mealtimeFound, type: "mealtime"}] : []
+}
```

Comment:
I would have done 
```suggestion
   for(mealTime of MealTimes){
     const formattedType = type.toLowerCase();
     if(mealTime.names.includes(formattedType)){
       return [{value: mealTime, type: "mealtime"}];
     }
   }
   return [];
```
if looking for only one occurrence as your function does (the last one to be exact)

This if we are looking for all occurrences
 ```suggestion
const formatMealTime = mealTime => mealTime.value

const matchingMealTimes = MealTimes.filter(mealTime => {
  const formattedType = type.toLowerCase();
  return mealTime.names.includes(formattedType);
})
if (!matchingMealTimes) {
  return []
}

return matchingMealTimes.map(formatMealTime);
```